operators/aqua/2022.4.14/manifests/aqua-operator.v2022.4.14.clusterserviceversion.yaml

```
@@ -0,0 +1,731 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: aqua-operator.2022.4.14
```

@adishaull I think this is the problem:

```suggestion
  name: aqua-operator.v2022.4.14
```

---

Thank, fixing